modules/apps/export-import/export-import-service/src/main/java/com/liferay/exportimport/internal/data/handler/BatchEnginePortletDataHandler.java

```diff
@@ -286,6 +293,10 @@ protected String doExportData(
 					_executeExportTask(
 						Integer.MAX_VALUE, portletDataContext, registration);
 
+				if (result == null) {
```

Comment:
Is this necessary? I think the portletDataHandler that returns null, if active, it is because there is something wrong with the export. I am asking this because I don't understand this after invoking the isActive method

Reply:
It remained here from a previous version before conflict resolving. I'm going to remove this.